### PR TITLE
Fix-badges-and-remove-links-to-old-documentation

### PR DESCRIPTION
**Pull request by ni-jfitzger**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- ~[ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~
- ~[ ] I've added tests applicable for this pull request~

### What does this Pull Request accomplish?
* Remove nimi-python docs status
* Fix Travis CI Status Badge
* Replace Coveralls Status Badge with Codecov Status Badge
* Add status badges for new Read the Docs projects
  - Clicking on these will take users to the **latest** documentation
* Update Documentation README section to give links to **stable** documentation of indiividual Read the Docs projects

Based upon #2002

### List issues fixed by this Pull Request below, if any.
None

### What testing has been done?
Tested in original PR, though this has some modifications.
I tested the new links/badges in the overall status section.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -91,6 +89,11 @@ NI-DCPower Python API Status
     :target: http://pypi.python.org/pypi/nidcpower
 
 
+.. |nidcpowerDocs| image:: https://readthedocs.org/projects/nidcpower/badge/?version=latest
+    :alt: NI-DCPower Documentation Status
```
Comment (marcoskirsch) [inline README.rst:93]: consider 

NI-DCPower Python Bindings Documentation Status

or

NI-DCPower Python API Documentation Status

or

`nidcpower` Python module Documentation Status.

In short, make it so it is specific to Python. Use terminology that is consistent with whatever we use elsewhere.

File: docs/_static/documentation.inc [hunk 2]
```diff
@@ -3,6 +3,16 @@
 Documentation
 =============
 
-Documentation is available `here <http://nimi-python.readthedocs.io>`_.
+Documentation is available on **Read the Docs**:
+
+- https://nidcpower.readthedocs.io/en/stable
```
Comment (marcoskirsch) [inline docs/_static/documentation.inc:8]: Is this generated?
Seems like a shame to have to enumerate modules manually.

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (ni-jfitzger) [reply README.rst:93]: The section headings here say "... Python API Status" so Python API Documentation Status seems appropriate.

File: docs/_static/documentation.inc (hunk 2 above)
Comment (ni-jfitzger) [reply docs/_static/documentation.inc:8]: >Is this generated?

Not yet.

Comment (ni-jfitzger) [reply docs/_static/documentation.inc:8]: Maybe in the future we'll change this, but I'm not looking to make bigger decisions like that here.

**Comment by ni-jfitzger** [comment]
The codecov status badge numbers make me sad.

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**